src/frontend/src/components/LoggedInHeader/index.tsx

```diff
@@ -21,7 +22,10 @@ class LoggedInHeader extends React.Component {
               Account
             </button>
             <LogOut>
-              <button className="link hover-blue f6 f5-ns dib pa3">
+              <button
+                onClick={() => history.push('/')}
+                className="link hover-blue f6 f5-ns dib pa3"
+              >
```

Comment:
If we use React Router's Link Component, won't it update the history automatically? That seems like a better solution. Let me know if there's something I'm missing as to why you didn't go that route. I haven't used React Router before, I'm just going off the docs at https://reacttraining.com/react-router/web/guides/quick-start/example-basic-routing

Reply:
I had intended to wrap the button in the Link component but was cautioned against it here:

https://stackoverflow.com/questions/42463263/wrapping-a-react-router-link-in-an-html-button

as nesting it that way is not valid HTML.

Reply:
Hmm, okay. In that case, could you leave a comment linking to that stackoverflow post and explaining your rationale as you just did?